practices/reduce-coupling-between-abstractions.md

```diff
@@ -86,3 +94,24 @@ Clear abstractions and minimal dependencies support better documentation and col
 * Have we identified the most problematic areas of coupling in our codebase?
 * What small, incremental changes can we make to start reducing coupling in these areas?
 * How do we ensure system stability while refactoring to reduce coupling?
+
+
+### [Do a Spike](/practices/do-a-spike.md)
+
+<ed> This may be a silly question, but does your target reader know what a spike is? I'm assuming they do and we don't need to define this practice, but just want to double-check. </ed>
```

Comment:
I would guess they do. We can update the name to something like, "timebox an experiment" or something.

Reply:
Ok, I will add this clarification to the running text.